specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
About the second comment, have you actually tried this? Because I created those folders and put a file there and ran that command and it worked. The only strange thing was that the table name that was created was `specs/data/transactions/may/transactions` instead of just `transactions`